dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Abstractions/ReflectionHelper.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ReflectionHelper.cs
+using System.Reflection;
+using Google.Protobuf;
+using Google.Protobuf.Reflection;
+using Microsoft.AutoGen.Contracts;
+using Microsoft.AutoGen.Core;
+
+namespace Microsoft.AutoGen.RuntimeGateway.Grpc.Abstractions;
+public sealed class ReflectionHelper
```

Comment:
I'm not sure i understand the point of this class. Clients should send this information as they register

Reply:
it can move to tests

Reply:
moved